File: web/src/app/core/models/sales/sales-order-header-summary.ts
export interface SalesOrderHeaderSummary {
  salesOrderId?: number;
  orderDate?: any;
  status?: number;
  statusDescription?: string;
  accountNumber?: string;
  address?: string;
  subTotal?: number;
  taxAmt?: number;
  freight?: number;
  totalDue?: number;
  empployeeFullName?: string;
}

File: api/src/main/kotlin/com/wilber/adventureworks/employeemanagement/domain/dto/sales/SalesOrderHeaderSummary.kt
package com.wilber.adventureworks.employeemanagement.domain.dto.sales

import java.math.BigDecimal
import java.time.LocalDateTime

class SalesOrderHeaderSummary(
    var salesOrderId: Int,
    var orderDate: LocalDateTime,
    var status: Int,
    var accountNumber: String?,
    var address: String,
    var subTotal: BigDecimal,
    var taxAmt: BigDecimal,
    var freight: BigDecimal,
    var totalDue: BigDecimal,
    var personFirsName: String,
    var personLastName: String
)
